modules/planning/planning.cc

```diff
@@ -163,10 +163,6 @@ Status Planning::Start() {
 
 void Planning::OnTimer(const ros::TimerEvent&) {
   RunOnce();
-  if (frame_) {
```

Comment:
FrameHistory is used at em_planner.cc line 172.

Reply:
I guess we should only depends on the previous trajectory, not previous frame. Actually many information in previous frame is invalid. I will remove this dependency tomorrow.